multitenancy-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenantEventProcessorControlServiceTest.java

```diff
@@ -97,7 +110,18 @@ private void mockConnectionManager() {
         when(connectionTenant2.adminChannel()).thenReturn(adminTenant2);
         ArgumentCaptor<String> contextCapture = ArgumentCaptor.forClass(String.class);
         when(axonServerConnectionManager.getConnection(contextCapture.capture()))
-                .thenAnswer(a -> contextCapture.getValue().equals("tenant-1") ? connectionTenant1 : connectionTenant2);
+                .thenAnswer(a -> {
```

Comment:
Instead of using an ArgumentCaptor, you can just get the actual arguments from the `a` parameter instead. It makes it easier to read the test.